models/PyCryptoBot.py

```diff
@@ -397,54 +399,63 @@ def getSmartSwitchDataFrame(self, df: pd.DataFrame, market, granularity: int, si
             try:
                 df_first = None
                 df_last = None
-                
-                Logger.debug("Row Count (" + str(granularity) + "): " + str(df.shape[0]))
+				
+                #Logger.debug("Row Count (" + str(granularity) + "): " + str(df.shape[0]))
+
                 df_first = self.getDateFromISO8601Str(str(df.head(1).index.format()[0]))
                 df_last = self.getDateFromISO8601Str(str(df.tail(1).index.format()[0]))
+									 
             except Exception:
                 result_df_cache = pd.DataFrame()
-           
-            if (df_first is None and df_last is None) or (df_first.isoformat(timespec='milliseconds') > simstart.isoformat(timespec='milliseconds')):
-                df1 = self.getHistoricalData(market, granularity)
+
+            if (df_first is None and df_last is None):
+                Logger.info('--------------------------------------------------------------------------------')
+                Logger.info('             *** getting smartswitch (' + str(granularity) + ') market data ***')
+                Logger.info('--------------------------------------------------------------------------------')
```

Comment:
I'd rather not use those horizontal dividers -- it just increases the clutter.

Reply:
I only added them so it was more obvious when the bot was pulling data from the exchange, it should only appear at the start or near the beginning of a sim run, was just so the user could see something was happening as depending on the selected date range it can take a short while to return all the required data.

I can remove if that's preferred?

Reply:
If you think it's useful, please, keep them. Sorry for nitpicking.